src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -36,6 +36,28 @@
  * <pre>
  * &lt;module name=&quot;InnerTypeLast&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *   public void test1() {} // ok
+ *   class InnerTest1 {}
+ *   private String s; // violation, field should be declared before inner classes.
+ *
+ *   class InnerTest2 {
+ *     public void innerTest() {
+ *       int n; // ok
+ *       class LocalClass {
+ *         public void test3() {} // ok
+ *       }
+ *       int x; // ok
+ *     }
+ *   }
+ *
+ *   public void test() {} // violation, method should be declared before inner classes.
+ *   interface InnerInterface {}
+ *   private int n; // violation, field should be declared before inner classes.
```

Comment:
`before inner classes` it is not class above, but interface.
Please use "inner type" instead of class/interface, as check name says

Reply:
Done